docs/multicluster/getting-started.md

```diff
@@ -412,6 +412,91 @@ ResourceExport into the corresponding ResourceImport until users correct it.
 due to forementioned mismatch issue, Antrea Multi-cluster Controller will also skip converging
 the corresponding Endpoints ResourceExport until users correct it.
 
+## Multi-cluster ClusterNetworkPolicy Replication
+
+Antrea Multi-cluster admins can specify certain ClusterNetworkPolicies to be replicated across
```

Comment:
We should mention this feature is supported since Antrea version 1.6.0.

Reply:
Done